src/ListingPage/index.stories.tsx

```diff
@@ -241,84 +257,90 @@ interface PanelProps {
 
 const DetailsPanel = ({
   onClose = () => undefined,
 }: PanelProps): JSX.Element => (
-  <SlidePanel
-    header={<DetailsPanelHeader onClose={onClose} />}
-    content={<DetailsPanelContent />}
+  <Panel
+    onClose={onClose}
+    header={<DetailsPanelHeader />}
+    selectedTab={<DetailsPanelContent />}
+  />
+);
+
+const DetailsPanelWithTabs = ({
+  onClose = () => undefined,
+}: PanelProps): JSX.Element => (
+  <Panel
+    onClose={onClose}
+    header={<DetailsPanelHeader />}
+    tabs={[<Tab key="tab1" label="Tab 1" />, <Tab key="tab2" label="Tab 2" />]}
+    selectedTab={<DetailsPanelContent />}
   />
 );
 
 export const normal = (): JSX.Element => (
   <ListingPage
-    slidePanelOpen={false}
+    panelOpen={false}
     listing={listing}
-    filtersExpandable={false}
-    filters={<FiltersSummary />}
+    filters={<NonExpandableFilters />}
   />
 );
 
-export const normalWithOpenedPanel = (): JSX.Element => (
+export const withOpenPanel = (): JSX.Element => (
   <ListingPage
-    slidePanelOpen
+    panelOpen
     listing={listing}
-    filtersExpandable={false}
-    filters={<FiltersSummary />}
-    slidePanel={<DetailsPanel />}
+    filters={<NonExpandableFilters />}
+    panel={<DetailsPanel />}
   />
 );
 
-export const normalWithFiltersDetails = (): JSX.Element => (
+export const withOpenPanelAndTabs = (): JSX.Element => (
   <ListingPage
-    slidePanelOpen={false}
+    panelOpen
     listing={listing}
-    filtersExpandable
-    filters={<FiltersSummary />}
-    expandableFilters={<FiltersDetails />}
+    filters={<NonExpandableFilters />}
+    panel={<DetailsPanelWithTabs />}
   />
 );
 
-export const normalWithFiltersDetailsAndOpenedPanel = (): JSX.Element => (
+export const withExpandableFilters = (): JSX.Element => (
   <ListingPage
-    slidePanelOpen
+    panelOpen={false}
     listing={listing}
-    filtersExpandable
-    filters={<FiltersSummary />}
-    expandableFilters={<FiltersDetails />}
-    slidePanel={<DetailsPanel />}
+    filters={<ExpandableFilters />}
   />
 );
 
-export const normalWithFiltersDetailsAndOpenedIntegratedPanel = (): JSX.Element => {
+export const withFilterDetailsAndOpenPanel = (): JSX.Element => (
+  <ListingPage
+    panelOpen
+    listing={listing}
+    filters={<ExpandableFilters />}
+    panel={<DetailsPanel />}
+  />
+);
+
+export const withExpandableFiltersAndDynamicPanel = (): JSX.Element => {
   const [open, setOpen] = React.useState(true);
   return (
     <ListingPage
-      slidePanelOpen={open}
+      panelOpen={open}
       listing={listing}
-      filtersExpandable
-      filters={
-        <FiltersSummaryWithOpenSlidePanelButton onOpen={() => setOpen(true)} />
-      }
-      expandableFilters={<FiltersDetails />}
-      slidePanel={<DetailsPanel onClose={() => setOpen(false)} />}
-      slidePanelIntegrate
+      filters={<ExpandableFiltersWithOpenButton onOpen={() => setOpen(true)} />}
+      panel={<DetailsPanel onClose={() => setOpen(false)} />}
+      panelFixed={false}
```

Comment:
```suggestion
      panelFixed
```
A story with fixed panel is missing

Reply:
> I see a regression about Listing Page resizing when expanding or collapsing filters
> ![ListingPage is no longer resizing](https://user-images.githubusercontent.com/12515407/89504413-39b7cf80-d7c8-11ea-8ee2-6774e5c33b37.gif)

Can you elaborate on the regression? it seems to work as before for me

Reply:
Before Listing Page height was resized 'ouf of the box' (see : https://github.com/centreon/centreon-ui/blob/master/src/ListingPage/index.tsx#L60)
I will check on AutoDiscovery if it actually works